tests/queries/0_stateless/02477_age_via_unix_epoch.sql

```diff
@@ -0,0 +1,18 @@
+select 'year', age('year', toDate32('1969-12-25'), toDate32('1970-01-05'));
```

Comment:
There was a problem when `dateDiff` worked well with dates before and after Unix epoch. Please read #41978 So I fixed it in `dateDiff` function and created a test "via_unix_epoch" specially for that.
Maybe for `age` function such test is not needed. Because at least it's converted by test of `dateDiff`.  I may remove the test for `age` function.

Reply:
Removed 02477_age_via_unix_epoch test